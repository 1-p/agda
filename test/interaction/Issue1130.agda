-- {-# OPTIONS -v tc.with:40 #-}

id : (A : Set) → A → A
id A = {!id′!}
 -- C-c C-h produces:       id′ : ∀ {A} → A
 -- when it should produce: id′ : ∀ {A} → A → A
