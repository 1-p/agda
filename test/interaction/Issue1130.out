Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue1130 (Issue1130.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue1130.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : A → A\n" nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> (agda2-status-action "")
(agda2-info-action-and-copy "*Helper function*" "id' : ∀ {A} → A → A\n" nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> 